src/controllers/chatController.js

```diff
@@ -0,0 +1,67 @@
+import { Chat } from '../models';
```

Comment:
@gracelungu The controller's file name should be capitalized camel case as we are exporting a `Class`